unload/logic.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import os
+
+from django.apps import apps
+from django.conf import settings
+from django.template.loaders.app_directories import get_app_template_dirs
+
+
+class BaseSearch(object):
+
+    def get_templates(self, template_dir):
+        """
+        Get full paths to the app's templates.
+
+        :template_dir: full path to the app's template directory
+        :returns: a list of full paths to the app's templates
+        """
+
+        templates = []
+
+        # No template directories were found
+        if not template_dir:
+            return templates
+
+        for dirpath, dirnames, filenames in os.walk(template_dir):
+            for filename in filenames:
+                if filename.endswith('.html'):
```

Comment:
Files having `.html` extension is just a convention. In django templates can have any extension really. For instance you wouldn't put `.html` extension if you are creating a template for plain text email, you would use `.txt` most likely.